.github/workflows/services-json.yml

```diff
@@ -38,3 +41,57 @@ jobs:
           repo-token: "${{ secrets.GITHUB_TOKEN }}"
           title: "Service JSON Errors"
           input: "./validation_errors.json"
+
+  service_check:
+    name: Service Check
+    runs-on: ubuntu-latest
```

Comment:
If this job relies on any specific version packages, consider pinning the image at ubuntu-2004 or ubuntu-2204.

Reply:
Shouldn't cause any issues, the python setup action it uses work with 18.04, 20.04, and 22.04.

Reply:
It's mostly to keep a sudden change in two years from breaking things on us, but I won't block on this.